src/inst/opts/consul.asc

```diff
@@ -1,38 +1,16 @@
 === Custom Consul services
 
-Consul services check is based on defintion as per file `puppet-hpc/hieradata/common.yaml` extract:
+Consul services check is based on hash of hases hiera named : `profiles::consul::server::subservices` and defined
+ in file `$ADMIN/puppet-hpc/hieradata/common.yaml`:
```

Comment:
1/ You introduced a new typo with hases
2/ The sentence with 2 columns (`:`) is weird
3/ The wrapping is not correct
4/ You don't explain the purpose of the following yaml excerpt